### ISSUE TITLE
Instruction URL for `turso dev` is invalid

### ISSUE THREAD

**Issue by evccyr** [tursodatabase/turso-cli#783]
After running the `turso dev` command I was presented with this warning:
```
❯ turso dev
Could not start libsql-server.
o install it, follow the instructions at https://github.com/tursodatabase/libsql/blob/main/libsql-server/docs/BUILD-RUN.md
```
Invalid URL: https://github.com/tursodatabase/libsql/blob/main/libsql-server/docs/BUILD-RUN.md

**Comment by hrfmartins** [comment]
Do you have the latest version of the CLI? It seems this has been fixed, [#725](https://github.com/tursodatabase/turso-cli/pull/725))

**Comment by evccyr** [comment]
I had the AUR version installed. Closing.
